Algorithms.Tests/Sequences/DivisorsCountSequenceTests.cs

```diff
@@ -0,0 +1,31 @@
+using System.Linq;
+using System.Numerics;
+using Algorithms.Sequences;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Algorithms.Tests.Sequences
+{
+    public class DivisorsCountSequenceTests
+    {
+        [Test]
+        public void First10ElementsCorrect()
+        {
+            // Values taken from https://oeis.org/A000005 for comparison.
```

Comment:
```suggestion
            // These values are taken from https://oeis.org/A000005 for comparison.
```
Just to restart checks :)